packages/good-design/src/apps/ubi/context/ClaimContext.tsx

```diff
@@ -173,16 +173,22 @@ export const ClaimProvider: FC<PropsWithChildren<ClaimProviderProps>> = ({
 
       if (account && hasClaimed && !noUnclaimedPools && !loading) {
         await explorerPollLock.acquire("pollLock");
+        const poolContracts = [claimDetails.address, ...activePoolAddresses];
```

Comment:
@sirpy 
Poolcontracts cannot be used as they were before.
the ones used kept track of contracts to still claim from (as it is based on the `preClaimPools` passed down to useMultiClaim)

post-claim this array would have been always empty.

Pushed an alternative based on active-pool-addresses here:https://github.com/GoodDollar/GoodWeb3-Mono/pull/216/commits/d878dfbb2c0b2f7f15c452a966c65cbc6073e8e1